### DISCUSSION TITLE
How to active auto-import  "Text" Component?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#3966] [Q&A] [answered]
<img width="471" alt="image" src="https://user-images.githubusercontent.com/124666577/229010200-f3fe9d25-f946-40a9-a23e-04e0f5df012d.png">

**Comment by rtivital** [comment, marked as answer]
It cannot be done because Text is a native contructor

**Reply by vaynevayne** [reply]
Okay, well, in that case, could mantine change his name? like `<Span >` @rtivital

**Reply by rtivital** [reply]
You can use Box component instead of Text
